vir/src/context.rs

```diff
@@ -484,16 +484,32 @@ impl<'tcx> VirCtxt<'tcx> {
         })
     }
 
-    const fn get_int_data(ty: Type, rust_ty: &ty::TyKind) -> (u32, bool) {
-        assert!(matches!(rust_ty, ty::Int(_) | ty::Uint(_)));
-        let TypeData::Int { bit_width, signed } = *ty else {
-            unreachable!();
-        };
-        assert!(matches!(rust_ty, ty::Int(_)) == signed);
-        (bit_width as u32, signed)
+    const fn get_int_data(rust_ty: &ty::TyKind) -> (u32, bool) {
```

Comment:
Sorry, the previous comments were not clear: I meant to say that there is already a `rustc` API for getting the bit-width of a type, so let's use that. But this is also relatively small so not a strong opinion. @JonasAlaif ?